sdks/java/io/rrio/src/main/java/org/apache/beam/io/requestresponse/Call.java

```diff
@@ -100,7 +100,7 @@ Call<RequestT, ResponseT> ofCallerAndSetupTeardown(
             .build());
   }
 
-  private static final TupleTag<ApiIOError> FAILURE_TAG = new TupleTag<ApiIOError>() {};
```

Comment:
what is the reason this cannot be static now? Every call has a new failureTag instance created now.

Reply:
This caused errors that a PCollection could not be found by the FAILURE_TAG as it was being used multiple times. Every PCollection with a PCollectionTuple in the pipeline needed a unique TupleTag. This is the reason for removing the static final one.

Reply:
I see, does "unique TupleTag" here means they need to be unique reference (i.e. instance), so each PCollection instance holds a unique instance of TupleTag?

Probably good to add comment here. In case future trivial optimization do not have context and overwritten it

Reply:
Yes it seems that what you say is correct in that there needs to be a unique reference. I agree on adding the comment and will do.